sql/core/src/test/java/test/org/apache/spark/sql/JavaDataFrameSuite.java

```diff
@@ -354,7 +354,7 @@ public void testCountMinSketch() {
 
   @Test
   public void testBloomFilter() {
-    Dataset df = context.range(1000);
```

Comment:
Why does this code pass compile? Is it because we only turn scalac warnings to errors, but not javac warning?